### DISCUSSION TITLE
Highcharts Histogram (including other highcharts modules?)

### DISCUSSION THREAD

**Discussion by aFlocker** [zauberzeug/nicegui#1025] [Q&A] [answered]
### Question

I've been using Highcharts through NiceGUI for a while now for all sorts of types (bubble, scatter, line, etc..) but when I recently tried to use a '**histogram**' type, nothing would show up and I would see the following error:
![image](https://github.com/zauberzeug/nicegui/assets/43419109/32fc83e6-0a0c-4e2b-a3a8-cae5698d852f)

I think it's not recognizing that type because its not defined in the main ```highcharts.js``` but instead in a ```histogram-bellcurve.js``` file that needs to be imported. I notice I have a ```histogram-bellcurve.js``` in my _[project]/.conda/Lib/site-packages/nicegui/elements/lib/highcharts_modules/_ folder but for some reason it's not getting picked up?

I don't know much about how these web technologies work but I tried using ```ui.run_javascript()``` gave it a javascript call to "document.head.appendChild();" trying to add a script element pointing to the official Highcharts version of ```histogram-bellcurve.js``` ([https://code.highcharts.com/modules/histogram-bellcurve.js](https://code.highcharts.com/modules/histogram-bellcurve.js)) but that didn't seem to help...

So is there a good way for me to tell the browser or the niceGUI python package to include this file so that the '**histogram**' and '**bellcurve**' chart types can work?

Thanks!

**Comment by falkoschindler** [comment, marked as answer]
Hi @aFlocker!
You can use Highcharts extensions like "histogram-bellcurve" using the `extras` parameter:

```py
ui.chart({
    'series': [{
        'type': 'histogram',
        'data': [1, 2, 3, 4, 5],
    }]
}, extras=['histogram-bellcurve'])
```

**Reply by aFlocker** [reply]
ah, so easy! should've asked sooner ha
thanks so much!

**Comment by Achilles-96** [comment]
Is there an extras name for "organization" type as well?
